hostscripts/soc-ci/soc-ci

```diff
@@ -37,8 +37,8 @@ from six.moves import input
 
 # path to the user configuration file
 USER_CONFIG_PATH = os.path.expanduser('~/.soc-ci.ini')
-# possible mkcX hosts
-CI_WORKERS = list('abcdefghijklmnp')
+# possible mkcX hosts (there is no 'j' host)
+CI_WORKERS = list('abcdefghiklmnp')
```

Comment:
No. But the code tries to connect via ssh so it's annoying if you get a connection timeout...